### PR TITLE
SAN 3272 ttl for docker.events.publish queue

### PR DESCRIPTION
**Pull request by podviaznikov**
- [x] demo on gamma
- [x] merge hermes pr and update dependency
- [x] @anandkumarpatel

### CODE REVIEW

File: lib/rabbitmq.js [hunk 1]
```diff
@@ -33,6 +33,13 @@ var dockerEventPublishQueue = hostIp.replace(/\./g, '-') + '.docker.event.publis
 var tasks = {}
 tasks[dockerEventPublishQueue] = require('./workers/docker.event.publish.js')
 
+var dockerEventPublishQueueDef = {
+  name: dockerEventPublishQueue,
```
Comment (anandkumarpatel) [inline lib/rabbitmq.js:37]: this should be a string?

Comment (anandkumarpatel) [inline lib/rabbitmq.js:37]: nevermind, ignore above

File: package.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
     "morgan": "^1.5.0",
     "node-uuid": "^1.4.1",
     "ponos": "^1.3.0",
-    "runnable-hermes": "git://github.com/Runnable/hermes.git#v6.4.0"
+    "runnable-hermes": "git://github.com/Runnable/hermes.git#SAN-3272-ttl-per-queue"
```
Comment (anandkumarpatel) [inline package.json]: dont forget to update to tag before merge

**Comment by podviaznikov** [comment]
demoed on gamma to @anandkumarpatel

**Pull request merged**